source/developer/toolkit.rst

```diff
@@ -34,7 +34,7 @@ Below is a list of planned features of the developer toolkit with estimated time
  - The ability to build webapp client plugins to override existing UI components (replace posts with your custom components, use your own video services etc.), modify/extend client drivers to interact with custom server API endpoints, and add whole new UI views in predetermined places.
 
 3. Unscheduled:
- - The ability to build Go 1.8 server plugins to hook directly into server events (think new post events, user update events, etc.), have some form of database access (possibly access to certain tables, and the ability to create new tables) and to add custom endpoints to extend the Mattermost REST API.
```

Comment:
This entire list is dated. But it popped up in my search, so figured I could at least eliminate the Go/1.8 reference.